### DISCUSSION TITLE
Cold-start problem for GluonTS

### DISCUSSION THREAD

**Discussion by miteshyadav** [awslabs/gluonts#1474] [Q&A] [answered]
## Description
Is there any way to run inference for data with no historical time series value but just it's attributes. I know that GluonTS handles the cold start problem, but haven't seen any examples of how you actually do it

**Comment by lostella** [comment, marked as answer]
@miteshyadav given a predictor, you should be able to give it data containing an empty target field

```
"target": []
```
